.dockerignore

```diff
@@ -0,0 +1,8 @@
+# Ignoring git and cache folders
```

Comment:
We don't need the whole pipelines code in the image- you can also ignore `templates` and `ansible`.